### PR TITLE
Swagger Documentation for InterestRateChartSlabsApiResource.java

### PR DESCRIPTION
**Pull request by luckyman20**

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResourceSwagger.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Set;
+
+/**
+ * Created by Chirag Gupta on 08/12/17.
+ */
+final class InterestRateChartSlabsApiResourceSwagger {
+
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResourceSwagger.java]: private constructor missing

**Comment by luckyman20** [comment]
@Sanyam96 Made the Changes

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResource.java [hunk 2]
```diff
@@ -135,7 +122,10 @@ public String retrieveOne(@PathParam("chartId") final Long chartId, @PathParam("
     @POST
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String create(@PathParam("chartId") final Long chartId, final String apiRequestBodyAsJson) {
+    @ApiOperation(value = "Create a Slab", httpMethod = "POST", notes = "Creates a new interest rate slab for an interest rate chart.\n" + "Example Requests:\n" + "\n" + "interestratecharts/1/chartslabs")
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResource.java]: Creats a new interest rate slab for an interest rate chart.

Mandatory Fields
periodType, fromPeriod, annualInterestRate
Optional Fields
toPeriod and description
Example Requests:

interestratecharts/1/chartslabs

Comment (Sanyam96) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResource.java]: All the above details in the notes section

Comment (Sanyam96) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResource.java]: seperated by new line characters

**Review by Sanyam96 — CHANGES_REQUESTED** [review]

**Comment by luckyman20** [comment]
@Sanyam96 Made the Changes

**Review by Sanyam96 — APPROVED** [review]

**Pull request merged**